### PR TITLE
Refactoring Inline Styles

### PR DESCRIPTION
**Pull request by dylanesque**
-Refactored inline styles out of VisitPage.js and Predictions.js

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -48,6 +48,29 @@ body {
   padding: 8em 0em;
 }
 
+/* Predictions.js */
+.prediction-button {
```
Comment (ethanbb) [inline src/index.css]: Am I missing something or did `prediction-button` not get added as a class on the Button it's supposed to apply to?

Comment (dylanesque) [reply src/index.css]: Ok, snapshot has been updated this should hopefully be all good now?

**Comment by knod** [comment]
As well as the button class, I think you'll also need to resolve the conflicts. Let us know if you need a hand. If you go it alone, remember to incorporate the changes dev made.

**Comment by dylanesque** [comment]
I'll definitely need help untangling these, I wouldn't have touched predictions if I knew somebody else was currently making changes to it at the same time.

**Comment by knod** [comment]
Considering the... interesting... code I've been producing lately, I'm a little reluctant to throw stuff around in this room. I'll take a look anyway, but if anyone else wants to take a stab at it, that would also be great.

**Comment by knod** [comment]
Made a PR on your repo's branch. Do with it what you will!

**Pull request closed without merging**